backend/app/services/git_skill/utils.py

```
@@ -0,0 +1,535 @@
+                        name=skill_name,
+                        description=metadata.get("description", ""),
+                        version=metadata.get("version"),
+                        author=metadata.get("author"),
+                        display_name=metadata.get("displayName"),
+                        tags=metadata.get("tags"),
+                    )
+                )
+
+            except Exception:
+                # Skip invalid skills
+                continue
```

Ruff (0.14.14)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)